packages/principled-forms/src/field/index.ts

```diff
@@ -48,31 +40,7 @@ export function validate<T extends InputTypes, F extends Field<T>>(
     ? Validity.valid()
     : onInvalid(validities.find(Validity.isInvalid)!.reason); // at least one by definition
 
-  // TODO: Can we do this without `any`?
-  return construct(field.isRequired, {
-    eager: field.eager,
-    type: field.type,
-    validators: field.validators,
-    validity: newValidity,
-    value: field.value,
-  } as any) as any;
-}
-
-export function validateChange<T extends InputTypes, F extends Field<T>>(
```

Comment:
My take is that this is entirely unnecessary: we'll just require the callers to do the right thing and call `validate({ ...field, value: newValue })`. And in fact if they choose to mutate the original field instead, that's also fine, b/c we're going to hand them a new instance from `validate()`.